### PR TITLE
[16.0][ADD] base_user_effective_permissions

### PR DESCRIPTION
**Pull request by hbrunn**
This module shows administrators a combined view of access rights and record rules per model with expressions expanded. This is convenient for reviewing permissions.

image

### CODE REVIEW

**Review by thomaspaulb — APPROVED** [review]
Very nice feature

**Comment by bosd** [comment]
@hbrunn Can you please retrigger a runboat?

**Comment by thomaspaulb** [comment]
You can do it with

/ocabot rebase

14 okt. 2024 12:49:40 bosd ***@***.***>:

**Comment by bosd** [comment]
> You can do it with 

Yeah, but sadly the stupid bot doesn't listen to me. One needs to be the author or maintainer. 

Luckily the 🤖 listens to you.

File: base_user_effective_permissions/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
```
Comment (bosd) [inline base_user_effective_permissions/readme/USAGE.rst]:
```suggestion
To use this module, you need to be in developer mode and:
```

File: base_user_effective_permissions/i18n/.empty
Comment (bosd) [inline base_user_effective_permissions/i18n/.empty:1]: Do we need to keep the .empty files?

**Review by bosd — APPROVED** [review]
LGTM 👍 , Just some minor non-blocking questions/remarks.

File: base_user_effective_permissions/i18n/.empty
Comment (hbrunn) [reply base_user_effective_permissions/i18n/.empty:1]: no, thanks for the headsup

**Pull request merged**